python/integrationtest/dro_map_gen.py

```diff
@@ -36,3 +36,11 @@ def generate_dromap_contents(n_streams, n_apps = 1, det_id = 3, app_type = "eth"
                                   rx_iface=app, rx_mac=f"00:00:00:00:00:0{app}", rx_ip=f"0.0.0.{app}")
             source_id += 1
     return json.dumps(the_map.as_json(), indent=4)
+
+def generate_default_dropmap_file(filename, n_streams):
+    with open(filename, 'w+') as fp:
+        data = generate_dromap_contents(n_streams)
+        fp.write(data)
+        fp.flush()
+        fp.close()
+    return filename
```

Comment:
I've also made changes to the `python/integrationtest/dro_map_gen.py` to generate the dro map, and also some files to create default daq_conf_json and timing_config files so that the user running the test has the choice of including their own config files or using default ones

As it stands, all that needs to be supplied is a connections.xml file that indicates the hardware which is to be tested on. 

Is this process of the defaults being hard-coded a good idea or it be preferable if I implement something similar to what's in integrationtest_nanorc (the con_dict code) and then have the hard coded values live in the test, as in the case of https://github.com/DUNE-DAQ/hsilibs/blob/develop/integtest/iceberg_real_hsi_test.py